include/hw_unique_key.h

```diff
@@ -46,8 +46,24 @@ extern "C" {
 #endif
 
 #define HUK_SIZE_BYTES (HUK_SIZE_WORDS * 4)
-#define ERR_HUK_MISSING 0x15500
 
+/** @brief Error value when the hardware unique key is missing */
+#define ERR_HW_UNIQUE_KEY_MISSING               (0x16501)
+
+/** @brief Error value when writing the hardware unique key failed */
+#define ERR_HW_UNIQUE_KEY_WRITE_FAILED          (0x16502)
+
+/** @brief Error value when the generation of the hardware unique key failed */
+#define ERR_HW_UNIQUE_KEY_GENERATION_FAILED     (0x16503)
+
+/** @brief Error value when key derivation using the hardware unique key failed */
+#define ERR_HW_UNIQUE_KEY_DERIVE_FAILED         (0x16504)
+
+/** @brief Error value when the hardware unique key had a generic failure */
+#define ERR_HW_UNIQUE_KEY_GENERIC_ERROR         (0x16505)
```

Comment:
I suggest to prefix the errors with the library name instead of `ERR_`, e.g. `HW_UNIQUE_KEY_ERR_FOO`

Reply:
That makes sense, +1

Reply:
Good suggestion, I updated it.